src/app/rundown/components/scrollable-timeline/scrollable-timeline.component.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ScrollableTimelineComponent } from './scrollable-timeline.component'
+import { PartEntityService } from '../../../core/services/models/part-entity.service'
+import { RundownService } from '../../../core/abstractions/rundown.service'
+import { instance, mock } from '@typestrong/ts-mockito'
+import { SharedModule } from '../../../shared/shared.module'
+
+describe('FollowPlayheadTimelineComponent', () => {
```

Comment:
Mismatch between describe text and component being tested.

Reply:
Fixed.